components/brave_wallet/browser/eth_data_builder.cc

```diff
@@ -59,6 +59,42 @@ absl::optional<std::string> ChainIdToVersion(const std::string& symbol,
 
 }  // namespace
 
+namespace filforwarder {
+
+absl::optional<std::string> Forward(const std::vector<uint8_t>& destination) {
```

Comment:
This function's sole purpose is to convert FIL address to a contract call, so let's pass FilAddress here instead of bytes vector.